### PR TITLE
Added cosmos client bean sample to azure spring data cosmos repo

### PR DESCRIPTION
**Pull request by kushagraThapar**
Added cosmos client bean sample to azure spring data cosmos repo

### CODE REVIEW

**Review by trande4884 — APPROVED** [review]
LGTM

File: sdk/spring/azure-spring-data-cosmos/README.md [hunk 1]
```diff
@@ -622,6 +622,40 @@ private List<User> getUsersByLastName(String lastName, int pageSize) {
     return content;
 }
 ```
+### Using Azure Cosmos Client through Spring Data Cosmos
+- Azure-spring-data-cosmos supports using Azure Cosmos Client through Spring Data Cosmos. 
+- Users can get `CosmosClient` or `AsyncCosmosClient` bean through `ApplicationContext` and execute any operations supported by Azure Cosmos Client.
```
Comment (xinlian12) [inline sdk/spring/azure-spring-data-cosmos/README.md:627]:
```suggestion
- Users can get `CosmosClient` or `CosmosAsyncClient` bean through `ApplicationContext` and execute any operations supported by Azure Cosmos Client.
```

**Review by xinlian12 — APPROVED** [review]
LGTM, thanks

File: sdk/spring/azure-spring-data-cosmos/README.md [hunk 2]
```diff
@@ -622,6 +622,40 @@ private List<User> getUsersByLastName(String lastName, int pageSize) {
...
+        CosmosContainer myContainer = cosmosClient.getDatabase("myDatabase").getContainer("myContainer");
+        //  Creating a stored procedure
+        myContainer.getScripts().createStoredProcedure(
+            new CosmosStoredProcedureProperties("storedProcedureId", "function(){}"),
+            new CosmosStoredProcedureRequestOptions());
+        //  Reading a stored procedure
+        myContainer.getScripts().getStoredProcedure("storedProcedureId").read();
+    }
+
+    public void asyncCosmosClientBean() {
```
Comment (xinlian12) [inline sdk/spring/azure-spring-data-cosmos/README.md:647]: should we also align the name here?

Comment (xinlian12) [inline sdk/spring/azure-spring-data-cosmos/README.md:647]:
```suggestion
    public void cosmosAsyncClientBean() {
```

File: sdk/spring/azure-spring-data-cosmos/src/samples/java/com/azure/spring/data/cosmos/CosmosClientBeanCodeSnippet.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        CosmosContainer myContainer = cosmosClient.getDatabase("myDatabase").getContainer("myContainer");
+        //  Creating a stored procedure
+        myContainer.getScripts().createStoredProcedure(
+            new CosmosStoredProcedureProperties("storedProcedureId", "function(){}"),
+            new CosmosStoredProcedureRequestOptions());
+        //  Reading a stored procedure
+        myContainer.getScripts().getStoredProcedure("storedProcedureId").read();
+    }
+
+    public void asyncCosmosClientBean() {
```
Comment (xinlian12) [inline sdk/spring/azure-spring-data-cosmos/src/samples/java/com/azure/spring/data/cosmos/CosmosClientBeanCodeSnippet.java:33]: should we also align the name here?

Comment (xinlian12) [inline sdk/spring/azure-spring-data-cosmos/src/samples/java/com/azure/spring/data/cosmos/CosmosClientBeanCodeSnippet.java:33]:
```suggestion
    public void cosmosAsyncClientBean() {
```

**Pull request merged**